nvml/client_test.go

```diff
@@ -5,9 +5,9 @@ package nvml
 
 import (
 	"errors"
+	"github.com/hashicorp/nomad/helper/pointer"
```

Comment:
minor nitpick but can we `gomimports` this, so that it's separate from stdlib imports?

Reply:
my apologies and nice catch, pushed the requested changes